saleor/checkout/complete_checkout.py

```diff
@@ -248,7 +250,15 @@ def _create_order(*, checkout: Checkout, order_data: dict, user: User) -> Order:
     total_price_left = order_data.pop("total_price_left")
     order_lines = order_data.pop("lines")
 
-    order = Order.objects.create(**order_data, checkout_token=checkout.token)
+    site_settings = Site.objects.get_current().settings
```

Comment:
Small optimization - current site is available in `request.site`, it can be used in GraphQL mutation with `info.context.site`. We could pass it as param to save duplicated database query.

Reply:
@maarcingebala the location of that code line is to say the least.. complicated :)
in order to get to the request we would have to pass it in chain like:
`entry point` -> `complete_checkout` -> `_create_order`
and there are multiple entry points (`adyen webhooks` which would have to reach even higher for it, `checkout complete` mutation)
we sould save one query but add a lot of extra mess on the way

Reply:
That's exactly what we do to optimize e.g. checkout API in Saleor 3.0, where a lot of low-level functions fetch the plugin manager, while it is available in `info.context` in resolvers.

Reply:
Just a note, we should pass the site, not the entire request object.

Reply:
Added TODO note for now as checkouts are being refactored and this will have to be changed anyway. After further investigation that would require a lot of work (in adyen plugin) to make it site-aware just to pass it to complete_checkout.